iOS/SnapKit/PhotosExampleWithSnapkit/PhotosExampleWithSnapkit/ImageZoomViewController.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  ImageZoomViewController.swift
+//  PhotosExampleWithSnapkit
+//
+//  Created by 이조은 on 2022/09/13.
+//
+
+import UIKit
+import Photos
+
+class ImageZoomViewController: UIViewController, UIScrollViewDelegate {
+    
+    static let storyboardId = "imageZoomViewController"
+    
+    var asset: PHAsset! //전 화면에서 받아올 이미지
+    let imageManager: PHCachingImageManager = PHCachingImageManager()
+    
+    weak var imageView: UIImageView!
```

Comment:
weak으로 선언하신 이유와 타입을 UIImageView!로 써주신 이유가 궁금해요!

UIImageView() 객체가 imageView로 들어가지 않아서 아마 `self.imageView.image`로 접근할 때 런타임 오류가 뜰 것 같아요.

제가 추천하는 방법은 다음과 같습니다
```suggestion
    private let imageView = UIImageView()
```